spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -630,5 +630,52 @@ describe("kong.clustering.compat", function()
       assert.is_nil(assert(services[3]).ca_certificates)
     end)
 
-  end)
+  end)  -- describe
+
+
+  describe("route entities compatible changes", function()
+    it("route.snis contain wildcard", function()
+      for _, flavor in ipairs({"traditional", "traditional_compatible"}) do
```

Comment:
The for loop should include the describe block, like
```
for _, flavor in ipairs({ "traditional", "traditional_compatible"}) do
  describe("Router (flavor = " .. flavor .. ") route entities compatible changes", function()
```

Reply:
applied